### DISCUSSION TITLE
Update NPM with Theme.park

### DISCUSSION THREAD

**Discussion by Dvalin21** [tteck/Proxmox#3615] [Ideas]
Requesting to add to Nginx Proxy Manager Theme.park so users can choose a theme keep their eyes from burning out from the white back ground. https://github.com/themepark-dev/theme.park/tree/master/docker-mods

**Comment by Mellowlynx** [comment]
Duplicate of #3516

**Reply by Dvalin21** [reply]
Oops! closing
